### DISCUSSION TITLE
How to merge a tag with an existing tag?

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5482] [Q&A] [answered]
Also, create a rule to merge it automatically in future.

**Comment by Maista6969** [comment, marked as answer]
You can go into a tag and click the "Merge" button. This will add the other tag as well as all of its aliases to the merge target, which effectively "merges them in future"
![image](https://github.com/user-attachments/assets/28621381-fd73-46c7-b2ac-0b87c9efbbd2)
